### DISCUSSION TITLE
Search function for the starred list

### DISCUSSION THREAD

**Discussion by rml9ty** [community/community#52514] [Other Feature Feedback, Questions, & Ideas]
My list of starred projects is getting long and the only way to see what you are looking for is manually scrolling through it depending on when you added it to your starred list,

**Comment by ghost** [comment]
[]()

**Comment by sanxiyn** [comment]
Eh, no? Go to https://github.com/stars and there should be a search box with description "Search stars...".

**Reply by rml9ty** [reply]
Should have mentioned (my bad) I was talking about the iOS app.

**Comment by slmkhanahmed** [comment]
https://addons.mozilla.org/en-US/firefox/addon/git-star/
